pkg/skaffold/build/tag/tag_template.go

```diff
@@ -18,12 +18,72 @@ package tag
 
 import (
 	"bytes"
+	"errors"
 	"fmt"
 	"text/template"
 
 	"github.com/sirupsen/logrus"
 )
 
+// tagTemplateTagger implements Tagger
+type tagTemplateTagger struct {
```

Comment:
Would this cause any confusion with `envTemplateTagger` since they are both taggers that use templates? I was trying to maintain consistency by switching `env` with `tag`.

Reply:
ah yeah i see. i still think we should change it, i don't think it will cause any confusion

Reply:
Sounds good

Reply:
Sounds good. I'll keep this change to the backend.